setup.py

```diff
@@ -6,30 +6,16 @@
 Python Data Validation for Humans™.
 """
 
-from setuptools import setup, find_packages
 import os
-import re
 import sys
 
+from setuptools import find_packages, setup
 
 PY3 = sys.version_info[0] == 3
 HERE = os.path.dirname(os.path.abspath(__file__))
 
-
-def get_version():
```

Comment:
Why did you remove this function? I find this function really convenient in a sense that I don't have to remember to update the version number in two different places. If this is removed then I'd like to have a viable alternative for this rather than hard-coding the version number in this file as well.